www/templates/intro/consent.html

```diff
@@ -5,7 +5,7 @@
 <div class="consent-text">
 <h3>Introduction and Purpose</h3>
 This app is part of a travel behavior research platform developed at the
-University of California, Berkeley. The platform development is being led by <a
+Indian Institute of Science, Bengaluru. The platform development is being led by <a
```

Comment:
The platform was still developed at UC Berkeley. IISc is mainly customizing it. You can't say that the platform is being developed at IISc and then say that I am leading the development! I am a UCB grad student!